dockerfiles/evmapp/entrypoint.sh

```diff
@@ -200,9 +200,9 @@ if [ -z "${SCNODE_LOG_CONSOLE_LEVEL:-}" ]; then
 fi
 export SCNODE_LOG_FILE_LEVEL SCNODE_LOG_CONSOLE_LEVEL
 
-# History reset modifier status
-if [ "${SCNODE_HISTORY_RESET_MODIFIER_STATUS:-}" = "true" ]; then
-  HISTORY_RESET_MODIFIER_STATUS=true
+# setting resetModifiersStatus if provided
+if [ -n "${SCNODE_HISTORY_RESET_MODIFIER_STATUS:-}" ]; then
+  HISTORY_RESET_MODIFIER_STATUS="$(echo -en "        history {resetModifiersStatus = ${SCNODE_HISTORY_RESET_MODIFIER_STATUS}}")"
```

Comment:
I like this approach. Just one comment. This echo command will ouput the following

`
HISTORY_RESET_MODIFIER_STATUS='        history {resetModifiersStatus = false}'
`

If the config file will be happy with this format I would just use the following command for better aliment

`
HISTORY_RESET_MODIFIER_STATUS="$(echo -n "history {resetModifiersStatus = ${SCNODE_HISTORY_RESET_MODIFIER_STATUS}}")"
`

That will not add whitespaces before `history` word

Reply:
thanks, added!